test/integration/scheduler/plugins/plugins_test.go

```diff
@@ -55,21 +55,24 @@ import (
 
 // imported from testutils
```

Comment:
nit: remove this comment

Reply:
I'll follow this with another cleanup, thanks.